### PR TITLE
chore(backport): update unit tests after backport fo tendermint v0.35

### PR DESCRIPTION
**Pull request by shotonoff**
## Issue being fixed or feature implemented
After fixing compile errors, now the codebase is ready to fix the unit tests after backporting v0.35

## What was done?
Fix unit tests for backport of v0.35

## How Has This Been Tested?
Run unit tests

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

File: types/validator_set_test.go [hunk 1]
```diff
@@ -1268,77 +1264,6 @@ func TestNewValidatorSetFromExistingValidators(t *testing.T) {
 	assert.Equal(t, valSet.CopyIncrementProposerPriority(3), existingValSet.CopyIncrementProposerPriority(3))
 }
 
-func TestValSetUpdateOverflowRelated(t *testing.T) {
```
Comment (lklimek) [inline types/validator_set_test.go:1271]: why is this test removed?

File: types/block_test.go [hunk 2]
```diff
@@ -1254,20 +1252,6 @@ func TestCommit_ValidateBasic(t *testing.T) {
 			},
 			true, "commit cannot be for nil block",
 		},
-		{
-			"no signatures",
```
Comment (lklimek) [inline types/block_test.go:1258]: why is this test case removed?

**Review by lklimek — CHANGES_REQUESTED** [review]
Please also run `make lint` and fix issues.

**Review by shotonoff — COMMENTED** [review]
done

**Review by lklimek — APPROVED** [review]

**Comment by lklimek** [comment]
Squash-merge

**Pull request merged**